spec/feature/post_peep_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'capybara/rspec'
+
+feature "Posting peeps" do
+
+  before(:each) do
+    User.create(name: "Chidu", username: "Swagchief", email: "chidu@chidu.com", password: "12345")
+  end
+
+  scenario "Users are able to post peeps when signed in" do
+    expect(Peep.count).to eq(0)
+
+    sign_in("chidu@chidu.com", "12345")
+    peep("this is a test peep")
+    expect(Peep.count).to eq(1)
+
+    peep = Peep.get(1)
```

Comment:
Don't rely on the id of the newly created peep being 1.  Use `peep.first` instead.